scripts/promote-config.sh

```diff
@@ -36,9 +36,9 @@ main() {
     # except for manifest.yaml
     git checkout -- manifest.yaml
 
-    # also strip out the snoozes in the denylist because we don't want
-    # changes in the executed tests over time for production streams
-    sed -E -i 's/^(\s+)(snooze:\s+.*)/\1# \2 (disabled on promotion)/' kola-denylist.yaml
```

Comment:
Is there any specific reason `kola-denylist.yaml` was removed here?

It looks likes this is the reason for today's promote config failures: https://github.com/coreos/fedora-coreos-streams/actions/runs/6071168339/job/16468658157